go.mod

```diff
@@ -53,14 +53,11 @@ require (
 	github.com/emicklei/go-restful/v3 v3.8.0 // indirect
 	github.com/evanphx/json-patch v5.6.0+incompatible // indirect
 	github.com/evanphx/json-patch/v5 v5.6.0 // indirect
-	github.com/fatih/color v1.12.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
-	github.com/go-bindata/go-bindata v3.1.2+incompatible // indirect
```

Comment:
I see go-bin data being removed, we use go-bin data at many places , I am not sure if we can remove this directly, and if we can, does the places  using go-bin data get an alternative ?

Reply:
Yep, this is just a consequence of updating `go-jose`. I just trusted `go mod` to do the right thing with the rest of the dep changes.